apax/model/builder.py

```diff
@@ -155,14 +166,43 @@ def build_energy_derivative_model(
             init_box=init_box,
             inference_disp_fn=inference_disp_fn,
         )
-        if self.config["n_shallow_ensemble"] > 0:
+        if (
+            self.config["ensemble"]
+            and self.config["ensemble"]["kind"] == "shallow"
+            and self.config["ensemble"]["n_members"] > 1
+        ):
+            log.info("Bulding ShallowEnsemble model")
             model = ShallowEnsembleModel(
                 energy_model,
                 calc_stress=self.config["calc_stress"],
+                force_variance=self.config["ensemble"]["force_variance"],
             )
         else:
+            log.info("Bulding Standard model")
             model = EnergyDerivativeModel(
                 energy_model,
                 calc_stress=self.config["calc_stress"],
             )
         return model
+
+    def build_ll_feature_model(
+        self,
+        scale=1.0,
+        shift=0.0,
```

Comment:
These are not used, are they?

Reply:
no. This was to keep the signature in line with the other model building methods. But I guess it's not needed.

Reply:
maybe add a comment and set them to None?